PhysicsTools/PatAlgos/python/producersLayer1/patCandidates_cff.py

```diff
@@ -36,6 +36,12 @@
 )
 patCandidates = cms.Sequence(patCandidateSummary, patCandidatesTask)
 
+from PhysicsTools.PatAlgos.producersHeavyIons.heavyIonJets_cff import *
+_patCandidatesTask = patCandidatesTask.copy()
+_patCandidatesTask.add(recoJetsHIpostAODTask)
```

Comment:
if `recoJetsHIpostAODTask` is meant to be done before or as a part of `makePatJetsTask`, please move its addition to PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py so that we do not mix internal details with a generic part of the setup